db/models/event.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+const db = require('../config.js');
+const Schema = mongoose.Schema;
+
+let eventSchema = new Schema({
+  title: {type: String, required: true},
+  description: {type: String, required: true},
+  planId: {type: Schema.ObjectId, ref: 'Plan', required: true },
+  startTime: {type: String, required: true},

```

Comment:
required? why? what are we defaulting this to?

Reply:
I was following the API page, we can edit this